apps/mobile/src/components/InteractiveLink.tsx

```diff
@@ -10,6 +10,7 @@ import { Typography, TypographyProps } from './Typography';
 export type InteractiveLinkProps = PropsWithChildren<{
   href?: string;
   showUnderline?: boolean;
+  isSecondary?: boolean; // if true, the text color renders as normal Typography..
```

Comment:
I think `isSecondary` is a misleading prop name because it suggests that there's are primary and secondary versions of InteractiveLinks, which we don't have.  (unlike Button.tsx where we do have primary and secondary buttons that are designed to look unique)

instead of adding a prop here for this use case let's stick to our current pattern of conditionally passing a different color to textStyles depending on dark/light mode

example
https://github.com/gallery-so/gallery/blob/a50d634bb05c31a1c888c97d16bca74d56048601/apps/mobile/src/screens/NftDetailScreen/NftAdditionalDetailsTezos.tsx#L53